app/controllers/charges_controller.rb

```diff
@@ -27,46 +25,40 @@ def create
 
   private
 
-  def create_order_for_vendors
-    unique_vendor_ids = {}
-    @order_ids = []
+  def send_notifications
+    notify_boss
+    email_customer
+  end
 
-    cart.cart_items.each do |cart_item|
-      unique_vendor_ids[cart_item.user.id] = 0
-    end
+  def email_customer
+    NotificationsMailer.customer_order(current_user, @order_ids).deliver_later
+  end
 
-    unique_vendor_ids.each_key do |vendor_id|
-      order = Order.create(
-        user_id: vendor_id,
-        status:  "ordered",
+  def create_order_for_vendors
```

Comment:
Assignment Branch Condition size for create_order_for_vendors is too high. [15.03/15]